### ISSUE TITLE
Vanilla demo does not work at all

### ISSUE THREAD

**Issue by dylanh724** [kylewetton/image-compare-viewer#34]
Using CDN and template init code:

![image](https://user-images.githubusercontent.com/8840024/135747156-0afc25ed-753c-48ef-84db-598e72c22127.png)

```js
const element = document.getElementById("image-compare");

    const options = 
    {
        // UI Theme Defaults      
        controlColor: "#FFFFFF",
        controlShadow: true,
        addCircle: false,
        addCircleBlur: true,
      
        // Label Defaults      
        showLabels: true,
        labelOptions: 
        {
          before: '1',
          after: '2',
          onHover: false
        },
      
        // Smoothing [looks laggy]      
        smoothing: false,
        smoothingAmount: 100,
      
        // Other options      
        hoverStart: false,
        verticalMode: false,
        startingPoint: 75,
        fluidMode: true
      };
        
      // Add your options object as the second argument
      const viewer = new ImageCompare(element, options).mount();
```

![image](https://user-images.githubusercontent.com/8840024/135747039-40677120-1e26-415e-a04b-c51ae88c545d.png)

Chrome desktop, latest, Win10.

Should I serve locally? Any known issue of this? Why is there even a jQuery (3.6.0) clash, I wonder?

**Comment by dylanh724** [comment]
Update: Same err with local instead of CDN

**Comment by dylanh724** [comment]
The problem is actually worse than I thought - opening a new ticket. The github demo doesn't work.
